convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -241,9 +247,19 @@ def __call__(self, patterns=None):
             pkg_obj.name = "installed_pkg"
             return [pkg_obj]
 
-    @unit_tests.mock(yum.rpmsack.RPMDBPackageSack, "returnPackages",
-                     ReturnPackagesMocked())
-    def test_get_installed_pkg_objects(self):
+    try:
+        @unit_tests.mock(yum.rpmsack.RPMDBPackageSack, "returnPackages",
+                         ReturnPackagesMocked())
+        def test_get_installed_pkg_objects_yum(self):
+            self.get_installed_pkg_objects()
+    except AttributeError:
+        # TODO: find out what we need to mock
+        @unit_tests.mock(yum.query.Query, "__repr__",
+                         ReturnPackagesMocked())
```

Comment:
Mocking out anything from `dnf.query.Query` throws an error. DNF internally uses hawkey/libdnf to handle packages but unsure how to fix this

```py
======================================================================
ERROR: test_get_installed_pkg_objects_dnf (convert2rhel.unit_tests.pkghandler_test.TestPkgHandler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/data/convert2rhel/unit_tests/__init__.py", line 68, in wrapped_fn
    setattr(class_or_module, orig_obj, mock_obj)
TypeError: can't set attributes of built-in/extension type '_hawkey.Query'
```

Reply:
Thanks, managed to find a solution!

That test seems to pass now, on to squash different bugs